src/AbstractValidator.php

```diff
@@ -81,7 +81,7 @@ public function canValidate(string $field): bool
       // feature, it can replace these messages with other data that's more
       // specific to its needs.
       
-      $messages[$field] = 'This field has a valid value.';
+      $this->messages[$field] = 'This field has a valid value.';
```

Comment:
it was always intended to be a reference to the $messages property, but I messed it up before this PR